ecmascript/ast/src/class.rs

```diff
@@ -102,6 +102,9 @@ macro_rules! property {
             #[serde(default)]
             pub readonly: bool,
 
+            #[serde(default)]
+            pub declare: bool,
```

Comment:
Should we get rid of this macro so this property doesn't appear on `PrivateProp`?

Reply:
I think it is right way to go

Reply:
Ok, sounds good. I'll do this now.

Also, how do I get the tests to automatically update the .json files? Previously it would just happen, but now I'm not sure how.

Reply:
You can do `UPDATE=1 cargo test`

Reply:
Sweet. Thanks!